openshift-template.yaml

```diff
@@ -100,6 +105,9 @@ parameters:
   s3_region_endpoint:
     type: string
     description: s3 region end point for registry backend
+  management_net:
+    type: string
+    default: "1b6888a0-75e5-489b-9757-4dcd95f86d09"
```

Comment:
Should we be hardcoding a default that will only ever work in a specific tenancy. Why not just remove it?